src/provider/provider_tracking.c

```diff
@@ -81,6 +81,16 @@ static umf_result_t umfMemoryTrackerRemove(umf_memory_tracker_handle_t hTracker,
 umf_memory_pool_handle_t umfMemoryTrackerGetPool(const void *ptr) {
     assert(ptr);
 
+    if (TRACKER == NULL) {
+        fprintf(stderr, "tracker is not created\n");
+        return NULL;
+    }
+
+    if (TRACKER->map == NULL) {
```

Comment:
When this condition is possible?

Reply:
The same like in https://github.com/oneapi-src/unified-memory-framework/pull/238#discussion_r1491558174.
- when `critnib_new()` fails because `umf_ba_linear_create()` fails